management-api-for-apache-cassandra-4.1.advisories.yaml

```diff
@@ -57,6 +65,10 @@ advisories:
             componentType: java-archive
             componentLocation: /opt/management-api/datastax-mgmtapi-server-0.1.0-SNAPSHOT.jar
             scanner: grype
+      - timestamp: 2024-07-01T07:24:57Z
```

Comment:
Could you elaborate why all these component are connected to swagger-jaxrs2 when the CVEs are related to different libraries e.g.  org.yaml:snakeyaml  ?

Reply:
these CVE's were already marked as Pending upstream fix in [management-api-for-apache-cassandra advisory](https://github.com/wolfi-dev/advisories/blob/main/management-api-for-apache-cassandra.advisories.yaml) This is a version stream, packaged was renamed.

Today before this PR I tried to fix these.
I have not checked all of them but these are mostly transitive dependency of the swagger-jaxrs2
```sh
[INFO] --- dependency:3.7.0:tree (default-cli) @ datastax-mgmtapi-server ---
[INFO] io.k8ssandra:datastax-mgmtapi-server:jar:0.1.0-SNAPSHOT
[INFO] \- io.swagger.core.v3:swagger-jaxrs2:jar:2.2.10:compile
[INFO]    \- org.yaml:snakeyaml:jar:2.0:compile
[INFO] --------------------------------------------
```
I have also tried use `<dependencyManagement>` to enforce `snakeyaml` transitive dependency version but getting the same error
```
2024/07/01 09:47:08 INFO [ERROR] COMPILATION ERROR : 
2024/07/01 09:47:08 INFO [ERROR] /home/build/management-api-server/src/main/java/com/datastax/mgmtapi/resources/LifecycleResources.java:[438,28] cannot access com.fasterxml.jackson.core.exc.StreamWriteException
2024/07/01 09:47:08 INFO   class file for com.fasterxml.jackson.core.exc.StreamWriteException not found
2024/07/01 09:47:08 INFO [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project datastax-mgmtapi-server: Compilation failure
2024/07/01 09:47:08 INFO [ERROR] /home/build/management-api-server/src/main/java/com/datastax/mgmtapi/resources/LifecycleResources.java:[438,28] cannot access com.fasterxml.jackson.core.exc.StreamWriteException
```

Reply:
didn't saw any CVE with score more than 8 so I didn't spend much time, and just copy pasted everything from the [management-api-for-apache-cassandra apackage advisory](https://github.com/wolfi-dev/advisories/blob/main/management-api-for-apache-cassandra.advisories.yaml)